yapf/yapflib/unwrapped_line.py

```diff
@@ -291,9 +291,9 @@ def _SpaceRequiredBetween(left, right):
       # If there is a type hint, then we don't want to add a space between the
       # equal sign and the hint.
       return False
-    if rval not in '[)]}.':
+    if rval not in '[)]}.' and not right.is_binary_op:
       # A string followed by something other than a subscript, closing bracket,
-      # or dot should have a space after it.
+      # dot, or a binary op should have a space after it.
       return True
```

Comment:
Loosing coverage on this line now. Suggestions for a test?

Reply:
It's weird, because the coverage should have gone up not down. It's "only" a 0.03% loss, so we can ignore it. But if the line is no longer being executed that's more troubling. Could you do a quick check to make sure that it is being executed when you run `python setup.py test`?

Reply:
The reason I loose coverage is because I made the conditional stricter. There are some `rval`s that were `not in '[)]}.'`, but they were all binary ops, which mean they don't hit the line now, even though they used to. So what we need are tokens that could conceivably come after a string but are not filtered by the new conditional. Suggestions?

Reply:
Ah! I read the patch wrong. This code looks like it's executed in "Chromium" style. There is a synthetic test like this:

```
b = '0' ('hello')
```

I say "synthetic" because I don't know why anyone would want to do something like this...but it looks like it's legal Python, so...

The test cases in `reformatter_basic_test.py` should be set up to use Chromium style by default.